### PR TITLE
Add queue details UI for admins

### PR DESCRIPTION
**Pull request by anbraten**
# Changes
- Adds an admin view to see the whole work-queue of the server. 
- The admin can also pause / resume the queue. 
- The view is reloading data every 5 seconds automatically.
- The task model from queue got removed in favor of the one from models.

image

### CODE REVIEW

File: server/router/api.go [hunk 1]
```diff
@@ -145,8 +145,8 @@ func apiRoutes(e *gin.Engine) {
 		{
 			queue.Use(session.MustAdmin())
 			queue.GET("/info", api.GetQueueInfo)
-			queue.GET("/pause", api.PauseQueue)
-			queue.GET("/resume", api.ResumeQueue)
+			queue.POST("/pause", api.PauseQueue)
```
Comment (6543) [inline server/router/api.go:148]: is breaking but :+1:

Comment (anbraten) [reply server/router/api.go:148]: I will add a note to the migration list

Comment (6543) [reply server/router/api.go:148]: need to adjust woodpecker-go too

Comment (6543) [reply server/router/api.go:148]: they are not used there so its fine & thanks for the docs change

Comment (anbraten) [reply server/router/api.go:148]: seems to be only having the info endpoint atm

**Review by 6543 — APPROVED** [review]

**Pull request merged**